torch/csrc/jit/passes/quantization.cpp

```diff
@@ -637,7 +637,9 @@ class InsertQuantDeQuantHelper {
   // TODO: we don't need to call this for each graph
   std::unordered_map<Graph*, std::vector<std::string>> observer_modules_to_remove_;
   std::unordered_map<Graph*, std::vector<Node*>> nodes_to_destroy_;
-  std::unordered_map<Graph*, std::unordered_map<Value*, QParamMap>> values_to_qparams_;
+  // Map from Graph to a map of output Value of forward call of observer to
+  // the original Value (input[1] of observer forward call)
+  std::unordered_map<Graph*, std::unordered_map<Value*, Value*>> values_to_quantize_;
```

Comment:
I don't quite understand why we need this Value-Value map - the observer call node has all this information already. The only use the observer output has is as a key in the call to `getQSchemeAndQParamMap` - we should just use the original value as the key and then we won't need this ObserverOutput->OriginalValue map.

Reply:
`getQSchemeAndQParamMap` uses observer output to get the observer node actually, it's harder to get observer node from original value since in that case observer node is one of the uses of the original value, although it should be the only use.
Maybe I can store observer node here, and observer output would be n->output() and original value would be n->input(1)?

Reply:
Yeah, that would be better.